src/routes/safe/store/actions/transactions/utils/transactionHelpers.ts

```diff
@@ -335,6 +363,10 @@ export const updateStoredTransactionsStatus = (dispatch, walletRecord): void =>
   }
 }
 
+enum PrimaryTypes {
```

Comment:
do we have more than one primary type? if no, why do we need an enum?

Reply:
well... the previous approach
```typescript
const primaryType: 'SafeTx' = 'SafeTx'
```

...was giving me the error:
```
ESLint: Expected a `const` assertion instead of a literal type annotation.(@typescript-eslint/prefer-as-const)
```

and that was the way I found to overcome the ts-lint error I was getting after setting `const primaryType = 'SafeTx'`

```
Types of property 'primaryType' are incompatible.
Type 'string' is not assignable to type '"SafeTx" | "EIP712Domain"'.
```

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-as-const.md

```
Examples of correct code for this rule:
...
let foo: 'bar' = 'bar' as const;
```

this looks like our case